### PR TITLE
Make the Lucene's merge policy parameters configurable by record cont…

### PR DESCRIPTION
**Pull request by tian-yizuo**
…ex's property storage

### CODE REVIEW

**Review by MMcM — DISMISSED** [review]

**Comment by tian-yizuo** [comment]
test this please

**Comment by tian-yizuo** [comment]
test this please

**Review by sophiabuell — APPROVED** [review]

**Comment by sophiabuell** [comment]
Jenkins test this please

File: fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneRecordContextProperties.java [hunk 1]
```diff
@@ -77,4 +77,24 @@
      * Call {@link RecordLayerPropertyKey#buildValue(Supplier)} with a supplier if you want to override this property with a value other than default.
      */
     public static final RecordLayerPropertyKey<Integer> LUCENE_SPELLCHECK_SEARCH_UPPER_LIMIT = RecordLayerPropertyKey.integerPropertyKey("com.apple.foundationdb.record.lucene.spellcheckSearchLimitation", 5);
+
+    /**
+     * Maximum segment size to produce during normal merging for ordinary full-text search with Lucene.
+     */
+    public static final RecordLayerPropertyKey<Double> LUCENE_MERGE_MAX_SIZE = RecordLayerPropertyKey.doublePropertyKey("com.apple.foundationdb.record.lucene.mergeMaxSize", 5.0);
```
Comment (alecgrieser) [inline fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/LuceneRecordContextProperties.java:84]: There should probably be units associated with this size and the other size-based parameter (I think it's in MB, but most of our API assumes bytes if there are sizes presented without units).

**Review by alecgrieser — APPROVED** [review]
I think it would be useful to put units on the size based parameters, but it otherwise LGTM

**Review by sophiabuell — APPROVED** [review]

**Pull request merged**